redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/AllocateDatabaseServerV4Request.java

```diff
@@ -1,10 +1,131 @@
 package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.requests;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.AwsDBStackV4Parameters;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.DatabaseServerV4Request;
+import com.sequenceiq.redbeams.api.endpoint.v4.stacks.NetworkV4Request;
+import com.sequenceiq.cloudbreak.common.mappable.Mappable;
+import com.sequenceiq.cloudbreak.common.mappable.ProviderParametersBase;
 
 import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 
 @ApiModel
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class AllocateDatabaseServerV4Request {
+public class AllocateDatabaseServerV4Request extends ProviderParametersBase {
+
+    // FIXME Define ModelDescriptions.DBStackModelDescription
+
+    @Size(max = 40, min = 5, message = "The length of the name has to be in range of 5 to 40")
```

Comment:
NIT: grammar

Reply:
I will address this in a follow-up change.